codbex-electra/opencart/listeners/sales-order-handler.js

```diff
@@ -0,0 +1,84 @@
+import { database } from "@dirigible/db";
+import { logging } from "@dirigible/log";
+
+const updateStatement = `
+	UPDATE oc_order 
+	SET
+		invoice_no = ?,
+		invoice_prefix = ?,
+		store_id = ?,
+		customer_id = ?,
+		comment = ?,
+		total = ?,
+		order_status_id = ?,
+		tracking = ?,
+		language_id = ?,
+		currency_id = ?,
+		accept_language = ?,
+		date_modified = ?
+	WHERE order_id = ?
+`;
+
+const logger = logging.getLogger("codbex-electra.opencart.listeners.sales-order-handler.js");
+
+exports.onMessage = function (messageString) {
```

Comment:
Could you make this export the ESM way?
```JS
export const onMessage = (messageString) => {
 ...
}
```
or
```JS
export function onMessage(messageString) {
 ...
}
```

Reply:
I like the seconds approach